modern/src/Theme.js

```diff
@@ -0,0 +1,50 @@
+import { createMuiTheme } from '@material-ui/core/styles';
+
+const traccarPurple = '#333366';
+const traccarGreen = '#4CAF50';
+const traccarWhite = '#FFF';
+
+export default createMuiTheme({
+  palette: {  
+    common: {
+      purple: traccarPurple,
+      green: traccarGreen
+    },
+    primary: {
+      main: traccarPurple
+    },
+    secondary: {
+      main: traccarGreen,
+      contrastText: traccarWhite
+    }
+  },
+  overrides: {
```

Comment:
We'll need to style the Form label as well. You can use the code below for that
```javascript
    MuiInputLabel: {
      filled: {
        transform: 'translate(12px, 14px) scale(1)',
          '&$shrink' :{
            transform: 'translate(12px, -12px) scale(0.75)'
          }
      },
    },
```

Reply:
Done.